src/main/java/me/karakelley/tictactoe/Application.java

```diff
@@ -0,0 +1,29 @@
+package me.karakelley.tictactoe;
+
+public class Application {
+
+  private GameLoop gameLoop;
+  private Validator playAgainValidator;
+  private UserInterface userInterface;
+  private String PLAY_AGAIN = "Do you want to play again (Y / N)?\n";
+
+  public Application(UserInterface userInterface, GameLoop gameLoop, Validator playAgainValidator) {
```

Comment:
I'm not crazy about the `Application` name. It's so broad and does not really tell me what this class is doing. Is it a 'GamesCoordinator', 'GamesReplay?`

Reply:
Yeah, I had a hard time coming up with a name, in hindsight Application doesn't really make much sense - the idea was that from here the game can be started and replayed again